### ISSUE TITLE
Loads, then goes to white

### ISSUE THREAD

**Issue by ghost** [johanw666/StarTrekTimelinesSpreadsheet#5]
Goes to white now. Anyone have any idea why?

**Comment by johanw666** [comment]
Yes, the client and API version have been updated. Client to 7.2.0, API to 14, see the new release instructions.

**Comment by ghost** [comment]
No that was me who said they had updated. This is an issue after it worked. Now it caches everything, then goes to "finishing up" then all white.

**Comment by johanw666** [comment]
You are right, I get the same. If they changed the API it will require reverse engineering the changes and recompilation I'm affraid.

**Comment by ghost** [comment]
That's part of what I'm not getting, as my API doesn't seem to be having any issues, at least it's not reporting anything to be different.

**Comment by FUNimations** [comment]
I've also noticed that. Tried some stuff. But it fails. However, everytime after reverting i'm able to use the new bundle.js for a while. Something that caught my eye today was in bundle.js you have
`i.CLIENT_PLATFORM="webgl"`
however when looking at the facebook webcalls being made i see:
```
platform: WebGLPlayer  <<<---- this one
device_type: Desktop
device_model: Chrome 75
device_mem_onboard: 512
gfx_device_id: 0
gfx_device_name: ANGLE (NVIDIA GeForce 210  Direct3D11 vs_4_1 ps_4_1)
gfx_vendor: WebKit
gfx_version: OpenGL ES 3.0 (WebGL 2.0 (OpenGL ES 3.0 Chromium))
gfx_memory: 512
gfx_shader_level: 35
cpu_processor_count: 1
cpu_processor_type: n/a
screen_size_inches: 9.655821
client_version: 7.2.0
platform_folder: webgl
client_api: 14
```
I've altered the platform parameter on my end just now. but as said, by having reverted, it always works for a while. But i'm also suspicious of the other parameters, i don't believe all thos parameters where submitted in the previous version.

**Comment by ghost** [comment]
IIRC, all of those have always been included, as I remember noticing it myself the very first time I learned how to see the API on facebook. 

With that said, it has been working for me without issues for the last few days. One of the new things I tried was to use a new token.

**Comment by johanw666** [comment]
If you have it working now, could you please describe the changes you made?

**Comment by ghost** [comment]
Only things I did was uninstall, reinstall then change both client and api versions in both documents to 7.2 and 14

**Comment by Inselmann2000** [comment]
Change it back to 7.1 and 12. The new version was rolled back. Way to much issues...

**Comment by ghost** [comment]
Wow, you're right, they reverted it. It's 7.1.0 and 13 btw
